governance/multisig_wh_message_builder/package.json

```diff
@@ -14,8 +14,7 @@
     "build": "tsc",
     "format": "prettier --write \"src/**/*.ts\"",
     "lint": "eslint src/",
-    "prepare": "npm run build",
```

Comment:
`prepare` will run on `npm ci` for the local packages even if I pass `--ignore-scripts` and this prevents us to have an optimized base lerna docker file.